### PR TITLE
Data objects changes

### PR DESCRIPTION
**Pull request by herokudev**
Create project without styling

Main functionalities:
Add books
Display existing books from local storage
Remove books

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @herokudev 👋

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  books = JSON.parse(localStorage.getItem('booksList'));
+  if (books !== null) {
+    books.forEach((book) => {
+      const div = document.createElement('div');
+      div.classList.add('myBook');
+      div.innerHTML = `
+    <h5>${book.title}</h5>  
+    <p>${book.author}</p>  
+    <button class="remove">remove</button>
+    <hr>
```
Comment (mikenath223) [inline main.js:20]: - [ ] After saving a book and refreshing the page the saved book disappears, kindly fix this function that listens to the DOMContentLoaded event, as localstorage should be used to persist the books array.

Comment (mikenath223) [reply main.js:20]: - It seems the books don't get saved to localstorage after adding a book. It should be a bug, please check it.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  addTitle.value = '';
+  addAuthor.value = '';
+});
+
+const removeBook = (elem) => {
+  if (elem.classList.contains('remove')) {
+    elem.parentElement.remove();
+  }
+};
```
Comment (mikenath223) [inline main.js:53]: - [ ] Also, removing a book functionality is broken it throws an error in the console, please check and fix it.

File: README.md [hunk 3]
```diff
@@ -2,75 +2,43 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Awesome books project.
```
Comment (mikenath223) [inline README.md:5]: - [ ] Please update the title `Project Name` to reflect the actual name of this project.

File: README.md [hunk 4]
```diff
@@ -2,75 +2,43 @@
...
 
 ![screenshot](./app_screenshot.png)
```
Comment (mikenath223) [inline README.md:7]: - [ ] Also, kindly update the project screenshot.

**Comment by Davidosky007** [comment]
## Hi @herokudev @mikenath223 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**